### ISSUE TITLE
Add support for new field in Application Gateway Web Application Firewall Policy settings

### ISSUE THREAD

**Issue by ssevugan21** [Azure/azure-cli#28598]
### Preconditions

- [X] No need to upgrade Python SDK or the Python SDK is ready.

### Resource Provider

Network Resource Provider (NRP)

### Description of Feature or Work Requested

We are adding support for customers to be able to configure cookie expiration duration in minutes for JavaScript Challenge feature within Application Gateway Web Application Firewall Policy settings. Field name is jsChallengeCookieExpirationInMins and the value can be an int32 with minimum as 5 and maximum as 1440

### Minimum API Version Required

2023-11-01

### Swagger PR link / SDK link

https://github.com/Azure/azure-rest-api-specs/pull/28011

### Target Date

2024-04-15

### PM Contact

sowmyam@microsoft.com

### Engineer Contact

ssevugan@microsoft.com

**Comment by yonzhan** [comment]
Thank you for opening this issue, we will look into it.

**Comment by ssevugan21** [comment]
@jsntcy  Is there a timeline to get this committed? Also please let know if you have any questions.

**Comment by ssevugan21** [comment]
@jsntcy @yonzhan any updates one this one?

**Comment by necusjz** [comment]
@ssevugan21 afaik, @ZengTaoxu is working on that

**Comment by ssevugan21** [comment]
@necusjz @ZengTaoxu  why is this being constantly postponed? I now see this being moved to June. Can we prioritize for May release? We are BLOCKED on the CLI changes and the only updates i see are for postponing the WI.

**Comment by necusjz** [comment]
@ssevugan21 i feel sorry about that, the previous milestone is quite short and called breaking change window--mainly for breaking changes. it caused we have limit bandwidth, actually, this work item has a predecessor task that has just been completed https://github.com/Azure/azure-cli/pull/28934.

after version bump, now we are working on that with highest priority. thanks for your understanding.
